src/Qowaiv.CodeAnalysis.CSharp/Rule.cs

```diff
@@ -42,14 +42,15 @@ public static partial class Rule
         title: "Characters with Trojan Horse potential are not allowed",
         message: "Trojan Horse character U+{0:X} detected.",
         description:
-            "When char",
+            "When characters with Trojan Horse potential are found, they should be blocked " +
+            "to avoid security risks to the application.",
         category: Category.Security,
         tags: ["Trojan", "Unicode"]);
 
     public static DiagnosticDescriptor SealClasses => New(
         id: 0005,
         title: "Seal concrete classes unless designed for inheritance",
-        message: "Seal this {0} or make it explicit inheritable.",
+        message: "Seal this {0} or make it explicitly inheritable.",
```

Comment:
I like this change! Just make sure that the test that assumes this message is updated accordingly.